converter/testdata/attributes-req-attrs.input

```diff
@@ -0,0 +1,18 @@
+[
+    {
+        "term": "noAttrs",
+        "definition": "Foobar",
+        "context": "",
+        "term_plural": "",
+        "reference": "",
+        "comment": ""
+    },
+    {
+        "term": "attrs",
+        "definition": "Foo {bar}",
+        "context": "",
+        "term_plural": "",
+        "reference": "",
+        "comment": ""
+    }
+]
```

Comment:
missing newline

Reply:
POEditor doesn't include newline in those files afair

Reply:
ah, right, sorry